lighthouse-core/audits/seo/link-text.js

```diff
@@ -44,6 +44,17 @@ const BLOCKLIST = new Set([
   'enlace',
   'este enlace',
   'empezar',
+  // Portuguese
+  'clique aqui',
+  'ir',
+  'início',
+  'inicio',
+  'ir',
+  'mais',
+  'mais informação',
+  'mais informações',
+  'veja mais',
+  'mais',
```

Comment:
duplicate of line 53, remove?

Reply:
ops. removing now and sorting alphabetically